qa/vagrant/src/test/resources/packaging/scripts/70_sysv_initd.bats

```diff
@@ -117,6 +117,17 @@ setup() {
     [ "$status" -eq 3 ] || [ "$status" -eq 4 ]
 }
 
+@test "[INIT.D] don't mkdir when it contains a comma" {
+    sed -i 's/DATA_DIR=.*/DATA_DIR=\/tmp\/foo,\/tmp\/bar/' /etc/init.d/elasticsearch
+    service elasticsearch start
+    wait_for_elasticsearch_status
+    assert_file_not_exist /tmp/foo,/tmp/bar
```

Comment:
I wonder if we should assert something a little stronger here? As if you've written, we shouldn't be creating any data directory now so maybe test that `/tmp/foo` and `/tmp/bar` do not exist?

But it also makes me wonder why we do not split on `,` and just create all the directories?